### PR TITLE
Add authorization variables to pvsite dev

### PR DESCRIPTION
**Pull request by simlmx**
Pass through the auth0 variables for pvsite dev.

### CODE REVIEW

**Comment by peterdudfield** [comment]
Should you merge to `development` first?

**Comment by simlmx** [comment]
@peterdudfield Indeed, I forgot about this!

**Review by peterdudfield — COMMENTED** [review]
Do you need to add them to the main variables.tf file too?

You might need to manually add them to terraform cloud too

**Comment by simlmx** [comment]
> Do you need to add them to the main variables.tf file too?
> 
> You might need to manually add them to terraform cloud too

They were already in there, and I set them in Terraform.

File: terraform/modules/services/api_pvsite/eb.tf [hunk 1]
```diff
@@ -61,6 +61,24 @@ resource "aws_elastic_beanstalk_environment" "eb-api-env" {
     value     = var.sentry_dsn
   }
 
+  setting {
```
Comment (peterdudfield) [inline terraform/modules/services/api_pvsite/eb.tf:64]: One other thing, need to add it to this too - https://github.com/openclimatefix/ocf-infrastructure/blob/main/terraform/modules/services/api_pvsite/docker-compose.yml

**Review by peterdudfield — APPROVED** [review]

**Pull request merged**